### PR TITLE
Adds the Motorhead random NPC class, audits scientist and doctor

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary
Content "Adds Motorhead NPC class and adjusts scientists and doctors"

#### Purpose of change
We could very much use more dynamic NPC classes with better, more reasonable starting skills and proficiencies. @John-Candlebury suggested that I add a race car enthusiast, who can possibly direct you to some unique things.

#### Describe the solution
This starts the process by creating a random NPC class that will eventually be able to give you that quest. If I get another break, I'll start adding the dialogue in this PR, but if not this can stand on its own.

In the process of reviewing skills and proficiencies to balance the class, I noticed some big problems in the skillset for a few others, most notably doctor and scientist. the errors were too egregious to ignore - Scientists weren't getting Chemistry! - so I audited them and, in the process, added proficiencies for the classes as well.

#### Describe alternatives you've considered
I considered differentiating "driver" and "mechanic" motorheads, but I think that's getting too specific for now. 
Later on, we might have a mechanic, just like we should probably develop different types of scientists.

#### Testing
I'm on  holiday and out of time, testing will have to wait.

#### Additional context
Vroom vroom.

### CODE REVIEW

**Comment by Paprika-H** [comment]
Why was the cowboy class removed?

File: data/json/npcs/NC_MOTORHEAD.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
+[
+  {
+    "type": "item_group",
+    "id": "NC_MOTORHEAD_pants",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "jeans", "prob": 70 },
+      { "item": "pants", "prob": 10 },
```
Comment (Paprika-H) [inline data/json/npcs/NC_MOTORHEAD.json:8]:
```suggestion
      { "item": "pants", "prob": 10 },
      { "item": "pants_leather", "prob": 25 },
      { "item": "motorbike_pants", "prob": 15 },
```

File: data/json/npcs/NC_MOTORHEAD.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  },
+  {
+    "type": "item_group",
+    "id": "NC_MOTORHEAD_coat",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "hoodie", "prob": 15 },
+      { "item": "jacket_light", "prob": 15 },
+      { "item": "jacket_windbreaker", "prob": 15 },
+      { "item": "jacket_leather", "prob": 20 },
```
Comment (Paprika-H) [inline data/json/npcs/NC_MOTORHEAD.json:47]:
```suggestion
      { "item": "jacket_leather", "prob": 20 },
      { "item": "jacket_leather_mod", "prob": 5 },
```

File: data/json/npcs/NC_MOTORHEAD.json [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+      { "item": "jeans", "prob": 70 },
+      { "item": "pants", "prob": 10 },
+      { "item": "pants_cargo", "prob": 20 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_MOTORHEAD_shoes",
+    "subtype": "distribution",
+    "entries": [ { "item": "sneakers", "prob": 60 }, { "item": "boots", "prob": 40 } ]
```
Comment (Paprika-H) [inline data/json/npcs/NC_MOTORHEAD.json:16]:
```suggestion
    "entries": [ { "item": "sneakers", "prob": 20 }, { "item": "boots", "prob": 60 }, { "item": "motorbike_boots", "prob": 30 }  ]
```

**Review by Holli-Git — COMMENTED** [review]
Adds leather pants, motorbike pants, motorbike boots, armored leather jacket to possible clothes to wear. Also adjusted shoes so boots were more common than sneakers

**Comment by I-am-Erk** [comment]
> Why was the cowboy class removed?

Misclick of some kind I suspect. Did this on my phone outside by the pool, surprised there aren't more. Should be able to fix it in a couple minutes.

**Comment by Paprika-H** [comment]
> Misclick of some kind I suspect. Did this on my phone outside by the pool, surprised there aren't more. Should be able to fix it in a couple minutes.

Subconscious cowboy bigotry

**Comment by I-am-Erk** [comment]
Damn, my biases are laid naked

**Comment by Paprika-H** [comment]
Also, "Motorhead" is slang for meth addict, would it make sense for these people to be on drugs, and interested in selling/buying alongside vehicle?

**Comment by ampersand55** [comment]
New game goal: recruit a motorhead named Lemmy and give him a handlebar mustache.

**Comment by I-am-Erk** [comment]
This is ready to test when I get home, or if someone wants to try in the meantime, cool.

**Pull request merged**